prow/lib.sh

```diff
@@ -105,3 +105,14 @@ function setup_e2e_cluster() {
   setup_cluster
 }
 
+function clone_cni() {
+  # Clone the CNI repo so the CNI artifacts can be built.
+  if [[ $PWD = ${GOPATH}/src/istio.io/istio ]]; then
+      TMP_DIR=$PWD
+      cd ../ || return
+      git clone -b master "https://github.com/istio/cni.git"
+      cd "${TMP_DIR}" || return
+  else
+    exit 1
```

Comment:
why is this condition needed? and this probably should spit out a more readable error message (if this indeed can happen?)

Reply:
I will change this.